core/templates/dev/head/domain/question/QuestionObjectFactorySpec.js

```diff
@@ -147,8 +147,9 @@ describe('Question object factory', function() {
       'id', 'name', 'notes', 'feedback');
     var misconception2 = MisconceptionObjectFactory.create(
       'id_2', 'name_2', 'notes', 'feedback');
-    expect(_sampleQuestion.validate([misconception1, misconception2])).toEqual(
-      'The following misconceptions should also be caught: name, name_2');
+    expect(
+      _sampleQuestion.validate([misconception1, misconception2])).toContain(
+        'The following misconceptions should also be caught: name, name_2');
```

Comment:
Shouldn't this include the extra "Click on ..." stuff?

Reply:
I used `toContain()` to match the string as the only part of the error message that has the misconception names in it is being matched in this test, while the rest of the string is constant for all error messages.

Reply:
Ah, I missed that, thanks. But can we make it toEqual() instead? It's a stronger check and we should have some check somewhere to ensure that the extra text is also shown.

Reply:
Ok, done.